docs/UserGuide.md

```diff
@@ -8,10 +8,17 @@ title: User Guide
 
 
 ## Introduction
+[<sub><sup>Back to top</sup></sub>](#table-of-contents)
+
 TAilor is a **desktop app for managing contacts, optimized for use via a Command Line Interface (CLI)**
 while still having the benefits of a Graphical User Interface (GUI). If you are a Computer Science Teaching
 Assistant who can type fast, TAilor can get tedious contact management tasks done faster than traditional GUI apps!
 
+This application is designed for Teaching Assistants of normal modules in NUS School of Computing. As such, only
```

Comment:
Hmm I understand where you're coming from too, but I'm not too sure if welcoming users through the requested introduction, only to turn them away in the "before we begin" is a good idea too?

If anything, I think we can include the requested additional sections under introduction, but not a separate section that is of the same level, what do you think?

Reply:
Hmm I see your point too. How about we keep our introduction only to information about TAilor on a whole (keeping it at a higher level) and add this information in the quick start as a caution box/call out or a subsection within? I think this information is too detailed/ at too low of a level to have in the introduction itself. What do you think?:)

Reply:
I think if its in the quick start, it might be even worse as the user should just be following steps to install and use our application, instead of checking if they fulfill the "requirements" to use the application?

Ie, if a line was included like:

- Check that you are a TA for a CS module in NUS.

That feels very offputting and if a person is not one, then they might feel cheated

Reply:
I still think its a good idea to have this in the introduction, possibly under a subheading.

I also do agree that the implementation does seem low level, so my proposed solution to that is something like:

```
### Notes on usage

TAilor is designed to be used by teaching assistants of normal modules in NUS School of Computing. Such modules include modules like CS2100 and CS2103T, but exclude DYOM modules. Currently, there is only support for students whose home university is NUS as well, so their matriculation numbers should be in the form of "A0123456X" as according to NUS.

If you do not fit the description above, don't worry, you still can use our application, but some features may seem a little odd to you. We are also currently working to produce a version that can be used by more people, so stay tuned!
```

This way, I omitted referring to the specific format requirements while still conveying the message about them

Reply:
I see:) So, two things-

I dont think we have a need to mention that this app is for TAs again, because we already state "If you are a Computer Science Teaching Assistant who can type fast," within the first 2 sentences of our introduction. What the phrasing can be is "TAilor currently supports only typical modules offered to students from the School of Computing. We do not cover DYOM modules at the moment as they do not conform to module naming conventions followed at SOC/ most NUS faculties. (etc etc)"

Secondly, coming to not wanting to put it in the quick start, I see your point too. So let's put this information in a subsection within the introduction, maybe titled "Things to note before you begin". I think that shall be the best way forward:))